import-export-cli/cmd/bundle.go

```diff
@@ -0,0 +1,136 @@
+/*
+*  Copyright (c) WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*  WSO2 Inc. licenses this file to you under the Apache License,
+*  Version 2.0 (the "License"); you may not use this file except
+*  in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+ */
+
+package cmd
+
+import (
+	"fmt"
+	"github.com/spf13/cobra"
+	"github.com/wso2/product-apim-tooling/import-export-cli/impl"
+	"github.com/wso2/product-apim-tooling/import-export-cli/utils"
+	"io"
+	"os"
+	"path/filepath"
+)
+
+var bundleDestination string
+var bundleSource string
+
+// Get command related usage Info
+const BundleCmdLiteral = "bundle"
+const BundleCmdShortDesc = "Archive any project to zip format"
```

Comment:
Shall we change this command description as follows to provide a more descriptive one?
```suggestion
const BundleCmdShortDesc = "Archive any Source project artifact to zip format"
```

Reply:
let's use like "Archive any **s**ource project artifact to zip format"

Reply:
Fixed with 9a12cbd